asm/lockController.asm

```diff
@@ -101,47 +101,154 @@ idle:
 	ijmp				; it is 0, do an indirect jump (address in Z register).
 
 
+; phase name convention:

```

Comment:
good convention! The sequence with just 1 bit ('e') is just a 'simple', or 'basic' sequence. The one that you called 'smooth', with 2 bits, is 'high-torque'. Let's call them
e -> elemental
s -> stronger  
So there's no need to change anything in the code.